src/FontDescriptor.h

```diff
@@ -1,13 +1,13 @@
 #ifndef FONT_DESCRIPTOR_H
 #define FONT_DESCRIPTOR_H
-#include <node.h>
-#include <v8.h>
-#include <nan.h>
-#include <stdlib.h>
-#include <string.h>
+
+#include <cstdlib>
+#include <cstring>
 #include <vector>
```

Comment:
This headers seem only necessary in `FontDescriptor.cc`, so they can be removed from here

Reply:
I never cleaned up the includes after splitting the files.